src/index.css

```diff
@@ -1,14 +1,16 @@
+@import url('https://fonts.googleapis.com/css?family=Muli');
```

Comment:
don't import fonts like this. It works but it loads slower than putting the `<link>` tag in the `<head>` tag of the html. This is because if you load it this way, the font file can't load until it's completely loaded and then parsed the index.css file. If you have it in the html, you make use of parallelism and have both the font and the css load at the same time making it asynchronous rather than synchronous.